build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Go to swift dir and build aperture
+cd swift
+xcodebuild
+
+# Move build artifact
+mv build/release/aperture main
+
+# Clean up
+rm -r build
```

Comment:
I would prefer to just keep this in package.json. It's so little code and it's unlikely to change.